activities.html

```diff
@@ -50,32 +50,30 @@
                 <h1>Activities</h1>
 
                 <h2>Reading Group</h2>
-                <h3>Tuesdays at 18:00 in BBB 3725</h3>
+                <h3>Wednesdays at 18:30 in BBB 3725</h3>
                 <p>
-                    Before each week's meeting, members read literature related to the week's topic. 
-                    At the meeting, a volunteer leader briefly presents, then guides discussion.
+                    Every Wednesday a speaker presents an AI/ML paper, and then as a group we discuss it. This is a great way to learn more about AI/ML in a low-stress environment. We are all here to learn and all are welcome!
+                    We cater food weekly from various restaurants. Before each week's meeting, members read literature related to the week's topic so then we can have an informed discussion on the topic.
                     See
-                    <a href="https://join.slack.com/t/msail-team/shared_invite/enQtMjUwMzc4NzUxMTQwLWVjMGMyMDMyYjFmOTgyZjU4MjlhZmQzODE0MzEyNjBmYWRiM2E3ZGQwZWZhNDM1N2E2YWVkMjIxMGI3ZjBiZTk">Slack</a>
+                    <a href="https://msail-team.slack.com/signup">Slack</a>
                     for more information!
                 </p>
 
                 <h2>Projects</h2>
-                <h3>Tuesdays and Sundays</h3>
                 <p>
-                    MSAIL engages in exploratory coding projects.  We hold group coding-brainstorming-(progress-updating) sessions
-                    weekly on Sundays.  We also hold office hours on Tuesdays at 19:00 in BBB 3725 and at other times by appointment
-                    to ensure project team members have the tools and direction they need.
-                    See
-                    <a href="https://join.slack.com/t/msail-team/shared_invite/enQtMjUwMzc4NzUxMTQwLWVjMGMyMDMyYjFmOTgyZjU4MjlhZmQzODE0MzEyNjBmYWRiM2E3ZGQwZWZhNDM1N2E2YWVkMjIxMGI3ZjBiZTk">Slack</a>
-                    for more information!
+                    MSAIL engages in exploratory coding projects. This semester we will be forming teams to work on cool AI topics ranging from natural language processing to computer vision to medical AI.
+                    MSAIL administration will pair your team with university faculty to facilitate a meaningful project experience that your team will then present at a poster symposium at the end of the semester.
+                    This is a great way to get an introduction to AI/ML, develop your coding skills, and gain real experience working on a team. If you are more experienced, we are also looking for project leads!
```

Comment:
We want to optimize for grad students as it will be fairly easy to recruit undergrad.

This is a great opportunity to further your research experience, explore AI/ML if you have not researched in the domain before, and present on your work!